docs/install/redhat.rst

```diff
@@ -28,11 +28,27 @@ from the CrateDB package repository::
     # Import the public GPG key for verifying the package signatures.
     sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
 
-    # Register with the CrateDB package repository.
-    sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/x86_64/crate-release-7.0-1.x86_64.rpm
-
-The command above will install the ``/etc/yum.repos.d/crate.repo`` package
-repository configuration file.
+To register with the CrateDB package repository, create a file called ``cratedb.repo``
+in the ``/etc/yum.repos.d/`` directory for RedHat based distributions, or in the
+``/etc/zypp/repos.d/`` directory for OpenSuSE based distributions, containing::
+
+    [cratedb-stable]
+    name=CrateDB RPM package repository - $basearch - Stable
```

Comment:
Using "CrateDB" and "cratedb" now instead of "CRATE"/"crate" like before. Also using an explicit label `cratedb-stable` for designating the stable package repository.